_config.yml

```diff
@@ -10,7 +10,9 @@ branch: 1.x
 #   {{site.webdev}}/downloads
 
 dartlang: ""
+dartlang_2: "https://dartlang-org-dev.firebaseapp.com"
 webdev: "https://webdev.dartlang.org"
+webdev_dev: "https://webdev-dartlang-org-dev.firebaseapp.com"
```

Comment:
Maybe `dev-webdev` (to follow the field naming contentions mentioned above, which use a hyphen rather than an underscore).